### PR TITLE
Create sign up modal

### PR DESCRIPTION
**Pull request by poom-kitti**
Test Page will be removed once done with testing.

Test Page can be accessed by clicking on "About One Pass" in navbar.

image

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -138,6 +140,7 @@ module.exports = {
       44: "22rem",
       48: "24rem",
       52: "26rem",
+      55.25: "27.625rem",
```
Comment (new5558) [inline tailwind.config.js]: ??

File: tailwind.config.js [hunk 2]
```diff
@@ -127,6 +128,7 @@ module.exports = {
       12: "6rem",
       14: "7rem",
       16: "8rem",
+      16.25: "8.125rem",
```
Comment (new5558) [inline tailwind.config.js]: ??

File: tailwind.config.js [hunk 3]
```diff
@@ -116,6 +116,7 @@ module.exports = {
       2.5: "1.25rem",
       3: "1.5rem",
       3.5: "1.75rem",
+      3.75: "1.875rem",
```
Comment (new5558) [inline tailwind.config.js]: ??

File: src/modules/global/modal-sign-up/ModalSignUp.vue [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        <h1 class="font-heading text-3xl">Sign up</h1>
+        <p class="text-gray-5">Powered by Chula SSO</p>
+      </section>
+      <section class="mb-2">
+        <h2>Student Email</h2>
+        <BaseTextInput class="w-full h-3.75" />
+      </section>
+      <section class="mb-4">
+        <h2>Password</h2>
+        <BaseTextInput class="w-full h-3.75" />
```
Comment (new5558) [inline src/modules/global/modal-sign-up/ModalSignUp.vue]: Why password is visible
image

File: src/modules/global/modal-log-in/ModalLogIn.vue [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+      <section class="mb-2">
+        <h1 class="font-heading text-3xl">Log in</h1>
+        <p class="text-gray-5">Powered by Chula SSO</p>
+      </section>
+      <section class="mb-2">
+        <h2>Student Email</h2>
+        <BaseTextInput class="w-full h-3.75" />
+      </section>
+      <section class="mb-4">
+        <h2>Password</h2>
```
Comment (new5558) [inline src/modules/global/modal-log-in/ModalLogIn.vue:13]: password visible

**Review by new5558 — CHANGES_REQUESTED** [review]

**Comment by new5558** [comment]
Will quickly implement storybook for frontend so that @poom-kitti will not have to create a test page every time he opened new PR

**Comment by poom-kitti** [comment]
I'm sorry I make a mistake. I added new component info-banner here instead of making a new branch.

File: src/App.vue [hunk 6]
```diff
@@ -17,3 +17,17 @@ export default defineComponent({
   }
 });
 </script>
+
+<style>
+.modal-authen-inner-container-width {
```
Comment (new5558) [inline src/App.vue]: why make it global variable?

**Comment by new5558** [comment]
Please attach an image preview of added components in the comments, so that the design team and I will know which component you are talking about

File: src/App.vue (hunk 6 above)
Comment (poom-kitti) [reply src/App.vue]: Because you say so.

Comment (new5558) [reply src/App.vue]: yes, for reusable. Its name don't sound reusable

**Review by new5558 — APPROVED** [review]

**Pull request merged**